applications/konami/src/module/konami_move.erl

```diff
@@ -93,13 +93,14 @@ get_originate_req(Data, Call) ->
                                  ,{<<"can_call_self">>, 'true'}
                                 ], Data),
 
-    OwnerId = wh_json:get_value(<<"owner_id">>, Data),
     {SourceDeviceId, TargetCallId} =
         case source_leg_of_dtmf(SourceOfDTMF, Call) of
             'a' -> {whapps_call:authorizing_id(Call), whapps_call:other_leg_call_id(Call)};
             'b' -> {find_device_id_for_leg(SourceOfDTMF), whapps_call:call_id(Call)}
         end,
 
+    DeviceOwnerId = cf_attributes:owner_ids(SourceDeviceId, Call),
```

Comment:
I think this type of code it going to be moved into the forthcoming `kazoo_objects` core lib, since it is being shared among whapps more frequently now. For now, I'd say leave it as-is so when we move the functionality to core, it is easy to identify what components are using it.